aiplatform/src/main/java/aiplatform/PredictCodeChatSample.java

```diff
@@ -0,0 +1,100 @@
+import com.google.cloud.aiplatform.v1beta1.PredictionServiceSettings;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+// Use a code chat model to generate a function
+public class PredictCodeChatSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String instance =
+        "{ \"messages\": [\n"
+            + "{\n"
+            + "  \"author\": \"content\",\n"
+            + "  \"content\": \"Hi, how are you?\"\n"
+            + "},\n"
+            + "{\n"
+            + "  \"author\": \"system\",\n"
+            + "  \"content\": \"I am doing good. What can I help you in the coding world?\",\n"
+            + " },\n"
+            + "{\n"
+            + "  \"author\": \"content\",\n"
+            + "  \"content\":\n"
+            + "     \"Please help write a function to calculate the min of two numbers.\",\n"
+            + "},\n"
+            + "],";
+    String parameters = "{\n" + "  \"temperature\": 0.2,\n" + "  \"maxOutputTokens\": 256,\n" + "}";
+    String project = "YOUR_PROJECT_ID";
+    String location = "us-central1";
+    String publisher = "google";
+    String model = "codechat-bison@001";
+
+    predictCodeChat(instance, parameters, project, location, publisher, model);
+  }
+
+  static void predictCodeChat(
+      String instance,
+      String parameters,
+      String project,
+      String location,
+      String publisher,
+      String model)
+      throws IOException {
+    String endpoint = String.format("%s-aiplatform.googleapis.com:443", location);
+    PredictionServiceSettings predictionServiceSettings =
+        PredictionServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (PredictionServiceClient predictionServiceClient =
+        PredictionServiceClient.create(predictionServiceSettings)) {
+      final EndpointName endpointName =
+          EndpointName.ofProjectLocationPublisherModelName(project, location, publisher, model);
+
+      // Use Value.Builder to convert instance to a dynamically typed value that can be
+      // processed by the service.
+      Value.Builder instanceValue = Value.newBuilder();
+      JsonFormat.parser().merge(instance, instanceValue);
+      List<Value> instances = new ArrayList<>();
+      instances.add(instanceValue.build());
+
+      // Use Value.Builder to convert parameter to a dynamically typed value that can be
+      // processed by the service.
+      Value.Builder parameterValueBuilder = Value.newBuilder();
+      JsonFormat.parser().merge(parameters, parameterValueBuilder);
+      Value parameterValue = parameterValueBuilder.build();
```

Comment:
nit: consider refactoring the code to make it more readable. For example:

```suggestion
List<Value> instances = new ArrayList<>();
instances.add(stringToValue(instance);
Value parameter = stringToValue(parameters);
// ... after line 100

static Value stringToValue(String value) {
  Value.Builder builder = Value.newBuilder();
  JsonFormat.parser().merge(value, builder);
  return builder.build();
}
```